src/tests/test_dynamic_api/test_temporal_actions.py

```diff
@@ -136,18 +136,21 @@ def test_details_record_can_be_requested_by_pk(self, api_client, stadsdelen):
     def test_details_default_returns_latest_record(self, api_client, stadsdelen):
         """Prove that object can be requested by identification
         and response will contain only latest object."""
-        url = reverse("dynamic_api:gebieden-stadsdelen-list")
-        response = api_client.get(f"{url}{stadsdelen[0].identificatie}/")
+        identificatie = stadsdelen[0].identificatie
+        url = reverse("dynamic_api:gebieden-stadsdelen-detail", args=(identificatie,))
+        response = api_client.get(url)
         data = read_response_json(response)
 
         assert response.status_code == 200, data
         assert data["_links"]["self"]["volgnummer"] == 2, data
+        assert data["id"] == stadsdelen[1].id, data
 
     def test_details_can_be_requested_with_valid_date(self, api_client, stadsdelen):
         """Prove that object can be requested by identification and date,
         resulting in correct for that date object."""
```

Comment:
Small nitpick:  "resulting in correct for" -> seems something is missing in that sentence.